devnode/Dockerfile

```diff
@@ -0,0 +1,43 @@
+FROM ubuntu:19.04
+
+LABEL maintainer="vincent_serpoul@manulife.com"
+
+RUN mkdir inst
+WORKDIR /inst
+
+COPY ./install_master_linux-amd64.tar.gz /inst/
+
+RUN tar -xf install_master_linux-amd64.tar.gz
+
+RUN apt-get update &&\
+    apt-get install -y ca-certificates
+
+RUN mkdir -p /root/node
+
+RUN ./update.sh -i -c stable -p ~/node -d ~/node/data -n
+
+######
+
+COPY ./genesis.json /root/randpool.genesis.json
+
+# Create randpool network
+RUN /root/node/goal network create -r /root/randpool -n private -t /root/randpool.genesis.json
+
+# activate telemetry
+RUN /root/node/diagcfg telemetry name -n randpool
```

Comment:
Actually private networks shouldn't generate telemetry (we still need to fix this in the code so they don't).  For now, please change this to `diagcfg telemetry disable`.